### DISCUSSION TITLE
input of Clint measurement for blood and liver?

### DISCUSSION THREAD

**Discussion by PriKalra** [Open-Systems-Pharmacology/Forum#828] [Questions & Problems] [answered]
Dear All, 

I have `Clint` measurements from `liver microsomes` (Unit: ml/min/mg) and also from `blood incubation` (Unit: ml/min)

I have incorporated the liver microsome clearance value as `metabolising enzymes -> In vitro metabolic rate in the presence of liver microsomes First order`

However, I am lost with how would I incorporate the value of blood incubation?

Best, 
Pri

**Comment by jzq90** [comment, marked as answer]
I think you can add a virtual enzyme in the animal/people page, set the expression only in the plasma or blood cells.
![Snipaste_2020-03-06_15-15-12](https://user-images.githubusercontent.com/58142266/76060899-505dbe80-5fbd-11ea-8967-ec83b90ddf47.jpg)

Then input the CLint in ‘metabolising enzymes -> In vitro metabolic rate in the presence of liver microsomes First order’. Select the virtual enzyme you just created. Keep the concentration in these two places equal.
